### PR TITLE
Adding Elliot For Water

### PR DESCRIPTION
**Pull request by gsambrotta**
Elliot For Water is an open source, ethical and sustainability friendly project. 
We have a core team of developers that can help and support contributors.

Let me know if the project doesn't meet your criteria somehow.
Thank you!

### CODE REVIEW

File: _data/projects/elliotForWater.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+site: https://ellitoforwater.com/
+tags:
+- water
+- search-engine
+- humanitarian
+- ethical
+- open-source
+- nextjs
+upforgrabs:
+  name: good first issues
```
Comment (shiftkey) [inline _data/projects/elliotForWater.yml]: I think this is the fix for the linter warning:

```suggestion
  name: good first issue
```

File: _data/projects/elliotForWater.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  link: https://github.com/ElliotForWater/elliotforwater.com/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (shiftkey) [inline _data/projects/elliotForWater.yml]: :art: a more succinct link

```suggestion
  link: https://github.com/ElliotForWater/elliotforwater.com/labels/good%20first%20issue
```

Comment (gsambrotta) [reply _data/projects/elliotForWater.yml]: ah, thanks @shiftkey

**Comment by ritwik12** [comment]
@gsambrotta Thanks!!

**Pull request merged**